Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <BaseOutputPath>$(MSBuildThisFileDirectory)binaries\</BaseOutputPath>
```

Comment:
We need to make sure that whatever we end up calling this path we add it to the `.gitignore`. For the record, regarding my other comment, `artifacts/` is already in the default `.gitignore`.